.github/workflows/main.yml

```diff
@@ -0,0 +1,71 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - master
+      - deploy
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  test:
+    name: Run tests
+    runs-on: ubuntu-latest
```

Comment:
Our prod server is on Ubuntu 16. 🙀 I don't think this is [an available runtime for GH Actions anymore](https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners). Any point in using the oldest OS that is available (Ubuntu 18)?